unic/ucd/normal/src/tables/canonical_combining_class_values.rsv

```diff
@@ -1,345 +1,345 @@
-// WARNING: Auto-generated by `tools/gen_ucd_tables.py`. DO NOT EDIT MANUALLY!
+// WARNING: Auto-generated by unic-gen. DO NOT EDIT MANUALLY!
```

Comment:
For this module at least this is feasible and I'm happy to do it. If we want to change the preamble in the python generated tables as well see #56, where I think would be the best place to apply that specific change.

If you want to just use the Python generated tables, change all the preambles with `unic-gen`, again, just tell me, I'll close #56 and fix up this and #59 to actually use the table generated by the Python scripts.

For the broader changes introduced to the `lookup_tables` tables, I'll have to see. That's a more involved change and I still don't _quite_ grep exactly what the code generating the maps to give to `emit_lookup_tables` is doing.

(To be frank, I'd also like to avoid unnecessarily writing Python2. Unless you are a legacy project you should be using Python3, there are no dependencies still stuck on Python2 that don't have a replacement in Python3. [Python2 is also retiring in 2020.](https://pythonclock.org/) [/tangent] (sorry))